src/client/app/components/admin/PreferencesComponent.tsx

```diff
@@ -229,6 +259,14 @@ class PreferencesComponent extends React.Component<PreferencesPropsWithIntl, {}>
 	private handleSubmitPreferences() {
 		this.props.submitPreferences();
 	}
+
+	private handleDefaultWarningFileSizeChange(e: { target: HTMLInputElement; }) {
+		this.props.updateDefaultWarningFileSize(parseFloat(e.target.value));
```

Comment:
I think these two parseFloat calls could cause an error. In other places in OED it checks the input value to be a valid float so maybe so that?

Reply:
I have set the input type for these two values to be number (lines 211 & 222). Do you think we still need to check the inputs?

Reply:
No, you are right and I missed this. You cannot enter a non-number into the box so it is safe.